lib/elixir/pages/Naming Conventions.md

```diff
@@ -0,0 +1,90 @@
+# Naming Conventions
+
+## Naming modules
+
+Elixir module names are written in "upper camel case", like `OptionParser`.
+
+Capital letters are kept in abbreviations, like `ExUnit.CaptureIO` or `Mix.SCM`.
```

Comment:
> No, I don't think we can because even trying to load the wrong module messes the VM up

This surprises me.  The error can't be rescued?  I'd like to learn a bit about this -- how does the Erlang VM get messed up when trying to load a non-existent module?

Even if nothing is done to make mix support `MyApp.CLI`, something should probably be done about the incredibly confusing error message mix provides:

```
** (Mix) The task "my_app.cli" could not be found. Did you mean "my_app.cli"?
```